examples/sklearn_autolog/pipeline.py

```diff
@@ -0,0 +1,44 @@
+from pprint import pprint
+
+import numpy as np
+from sklearn.linear_model import LinearRegression
+from sklearn.preprocessing import StandardScaler
+from sklearn.pipeline import Pipeline
+
+import mlflow
+
+
+def fetch_logged_data(run_id):
+    client = mlflow.tracking.MlflowClient()
+    data = client.get_run(run_id).data
+    tags = {k: v for k, v in data.tags.items() if not k.startswith("mlflow.")}
+    artifacts = [f.path for f in client.list_artifacts(run_id, "model")]
+    return {
+        "params": data.params,
+        "metrics": data.metrics,
+        "tags": tags,
+        "artifacts": artifacts,
+    }
+
+
+def main():
+    # enable autologging
+    mlflow.sklearn.autolog()
+
+    # prepare training data
+    X = np.array([[1, 1], [1, 2], [2, 2], [2, 3]])
+    y = np.dot(X, np.array([1, 2])) + 3
+
+    # train a model
+    pipe = Pipeline([("scaler", StandardScaler()), ("lr", LinearRegression())])
+    with mlflow.start_run() as run:
+        pipe.fit(X, y)
+
+    # show logged data
+    for key, data in fetch_logged_data(run._info.run_id).items():
+        print("-" * 10, key, "-" * 10)
```

Comment:
Nit: Can we print `"logged {key}" instead of "key"? E.g.,:

```
---------- logged params ----------
{'copy_X': 'True',
 'fit_intercept': 'True',
 'n_jobs': 'None',
 'normalize': 'False'}
---------- logged metrics ----------
{'training_score': 1.0}
---------- logged tags ----------
{'estimator_class': 'sklearn.linear_model._base.LinearRegression',
 'estimator_name': 'LinearRegression'}
---------- logged artifacts ----------
['model/MLmodel', 'model/conda.yaml', 'model/model.pkl']
```

Reply:
It'd be great if we could incorporate the run ID here as well